R/SS_RunJitter.R

```diff
@@ -26,7 +20,13 @@
 #' @param init_values_src Either zero or one, specifying if the initial values to
 #'   jitter should be read from the control file or from the par file, respectively.
 #'   The default is `NULL`, which will leave the starter file unchanged.
+#' @template exe
+#' @template verbose
+#' @param ... Additional arguments passed to r4ss::run(), such as
+#' `extras`, and `show_in_console`.
```

Comment:
no comma needed for a list of 2.

Reply:
good catch, fixed by adding forgotten 3rd element to list